### DISCUSSION TITLE
Provisioning profile problem with macos + xcode 13 beta + jb rider + ios

### DISCUSSION THREAD

**Discussion by okproject** [dotnet/maui#1736] [Q&A]
Hi. 
I am trying to use .net 6 Maui with my macos big sur.

- Following command works without problem: dotnet build HelloMaui -t:Run -f net6.0-maccatalyst --no-restore
- But command for ios, it throws error as following details. (provisioning profile problem)
- Problematic command: dotnet build HelloMaui -t:Run -f net6.0-ios --no-restore
Tools
- Xcode 13 Beta
- Jetbrains Rider 2021.1.2 EAP

- I tried to run maui-check and there is no problem
- I create a maui ios project with dotnet cli
- I installed ios simulator and i also have iphone with ios 14.6
- When i try to build project it throws exception during build process. Exception  looks like following
`
/usr/local/share/dotnet/packs/Microsoft.iOS.Sdk/15.0.100-preview.6.63/tools/msbuild/iOS/Xamarin.Shared.targets(914,3): error : Could not find any available **provisioning profiles** for HelloPreview6 on iOS. [/Users/rasuser/Documents/learning/ios/HelloPreview6/HelloPreview6/HelloPreview6.csproj]
/usr/local/share/dotnet/packs/Microsoft.iOS.Sdk/15.0.100-preview.6.63/tools/msbuild/iOS/Xamarin.Shared.targets(914,3): error :          [/Users/rasuser/Documents/learning/ios/HelloPreview6/HelloPreview6/HelloPreview6.csproj]
`

- Can you please share a solution for that ? I want to create ios project with dotnet cli and run it on iphone simulator or directly with my iphone. 
- I also tried install visual studio and add developer account to there; nothing fixed. I also test create a new project directly with xcode and run it on my iphone, there is no provisioning profile problem.
- As a summary, how can i fix provisioning profile  problem ?

**Comment by vfongmala** [comment]
Same here

**Reply by okproject** [reply]
Is there anyone  to check Q&A part in maui  repo ?

**Comment by okproject** [comment]
It looks like .net 6 & MAUI unattended;  even 20 days there, still no any answer.
Full of frustration.

**Comment by farbod1973** [comment]
I have the same problem:
/usr/local/share/dotnet/packs/Microsoft.iOS.Sdk/15.0.100-preview.7.230/tools/msbuild/iOS/Xamarin.Shared.targets(922,3): error : Could not find any available provisioning profiles ...

**Comment by AdamBebko** [comment]
Same issue
